app/models/chargeback/consumption_with_rollups.rb

```diff
@@ -95,6 +96,7 @@ def avg(metric, sub_metric = nil)
     end
 
     def current_value(metric, _sub_metric) # used for containers allocated metrics
+      resource = resource_type.classify.constantize.find(resource_id)
```

Comment:
@lpichler it's not obvious why this is necessary since `resources` was already being referenced below. Can you provide some detail?